core/services/vrf/integration_v2_test.go

```diff
@@ -917,6 +917,103 @@ func TestVRFV2Integration_SingleConsumer_NeedsTopUp(t *testing.T) {
 	assertNumRandomWords(t, consumerContract, numWords)
 }
 
+func TestVRFV2Integration_SingleConsumer_BigGasCallback_Sandwich(t *testing.T) {
+	config, db := heavyweight.FullTestDB(t, "vrfv2_singleconsumer_batch_bigcallback")
```

Comment:
Let's change the last argument to `FullTestDB` to reflect the name of the test - so remove the `batch` part and add `sandwich`.